### PR TITLE
Clean doc files and remove internal links

### PR DESCRIPTION
**Pull request by dshylov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
 ## The Python OJAI Client for MapR-DB
 
 A simple, lightweight library which provides access to MapR-DB.
-Client library able to supports all existing OJAI functionality and being absolutely capable with Java OJAI connector,
+Client library is able to support all existing OJAI functionality and being absolutely capable with Java OJAI connector,
 which running under MapR Data Access Service.
```
[the remark this answers is not in the document]
Comment (dshylov) [reply README.md]: Added

File: setup.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
                         'python-dateutil>=2.6.1', 'retrying>=1.3.3', 'future>=0.16.0'],
       python_requires='>=2.7.*',
       long_description='A simple, lightweight library which provides access to MapR-DB.'
-                       ' Client library able to supports all existing OJAI functionality'
+                       ' Client library is able to support all existing OJAI functionality'
                        ' and being absolutely capable with Java OJAI connector,'
                        ' which running under MapR Data Access Service.'
```
[the remark this answers is not in the document]
Comment (dshylov) [reply setup.py]: Added

File: setup.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
                         'python-dateutil>=2.6.1', 'retrying>=1.3.3', 'future>=0.16.0'],
       python_requires='>=2.7.*',
       long_description='A simple, lightweight library which provides access to MapR-DB.'
-                       ' Client library able to supports all existing OJAI functionality'
+                       ' Client library is able to support all existing OJAI functionality'
                        ' and being absolutely capable with Java OJAI connector,'
```
[the remark this answers is not in the document]
Comment (dshylov) [reply setup.py]: Added

File: README.md [hunk 4]
```diff
@@ -1,17 +1,18 @@
 ## The Python OJAI Client for MapR-DB
 
 A simple, lightweight library which provides access to MapR-DB.
-Client library able to supports all existing OJAI functionality and being absolutely capable with Java OJAI connector,
+Client library is able to support all existing OJAI functionality and being absolutely capable with Java OJAI connector,
```
[the remark this answers is not in the document]
Comment (dshylov) [reply README.md]: Added

File: docs/connection_instruction.md [hunk 5]
```diff
@@ -45,4 +45,4 @@ connection = ConnectionFactory.get_connection(connection_string)
 document_store = connection.create_store('/test-store)
 ```
 
-More examples how to use MapR-DB Python client you can find [here](https://github.com/mapr-demos/private-ojai-2-examples/tree/master/python)
\ No newline at end of file
+More examples how to use MapR-DB Python client you can find [here](https://github.com/mapr-demos/ojai-examples/tree/master/python)
```
Comment (NataliiaSkopiuk) [inline docs/connection_instruction.md]: Add dot at the end.

File: README.md [hunk 6]
```diff
@@ -1,17 +1,18 @@
 ## The Python OJAI Client for MapR-DB
 
-A simple, lightweight library which provides access to MapR-DB.
-Client library able to supports all existing OJAI functionality and being absolutely capable with Java OJAI connector,
-which running under MapR Data Access Service.
+A simple, lightweight library that provides access to MapR-DB.
+The client library supports all existing OJAI functionality and is absolutely compatible with Java OJAI connector,
+that runs under the MapR Data Access Gateway.
 
+The MapR official documentation [link](https://mapr.com/docs/61/MapR-DB/JSON_DB/UsingPythonOJAIClient.html)
```
Comment (NataliiaSkopiuk) [inline README.md]: Add dot at the end.

File: install_client.md [hunk 7]
```diff
@@ -1,24 +1,11 @@
...
-    trusted-host = artifactory.devops.lab
-    ``````
+3. Execute in terminal `pip install maprdb-python-client`
 
-5. Execute in terminal `pip install maprdb-python-client`
-
-Now you can use MapR-DB python client as a library.
\ No newline at end of file
+Now you can use MapR-DB python client as a library.
+More details and usage examples you can find in the [MapR official documentation](https://mapr.com/docs/61/MapR-DB/JSON_DB/GettingStartedPythonOJAI.html).
```
Comment (NataliiaSkopiuk) [inline install_client.md]: Please, change to: You can find more examples for using the MapR-DB Python client [here](https://mapr.com/docs/61/MapR-DB/JSON_DB/GettingStartedPythonOJAI.html).

Comment (dshylov) [reply install_client.md]: Changed

File: docs/connection_instruction.md (hunk 5 above)
Comment (dshylov) [reply docs/connection_instruction.md]: added

File: README.md (hunk 6 above)
Comment (dshylov) [reply README.md]: added

**Pull request merged**